packages/@okta/vuepress-theme-default/components/Sidebar.vue

```diff
@@ -67,6 +92,24 @@
           })
         }
         return false
+      },
+
+      expandSubNav: function (event) {
+        var parent = event.target.parentElement;
+        var sections = parent.querySelector('.sections');
+        if(sections === null) {
```

Comment:
I also generally argue in favor of using truthy/falsy values where you can - it keeps the focus of attention on the variable, not on `=== null` or whatever.

any reason this can't be `if( !sections ) { `?